README.md

```diff
@@ -28,6 +28,8 @@ Templates to help you start building on Omni.
 Practical examples and tutorials to understand how to work with Omni.
 
 - [Cross Chain Staking](https://github.com/omni-network/cross-stake) - Example cross chain app for multi-rollup staking with global state managed by the Omni EVM.
+- [XRegistry Contract](https://github.com/omni-network/omni/blob/main/contracts/src/protocol/XRegistry.sol) - Omni Registry contract that saves deployed contracts according to their chains for all supported rollups. Can be used as a utility for contract whitelisting.
```

Comment:
This description is a little confusing. Something more like "A contract address registry that syncs registrations across [XRegistryReplicas](https://github.com/omni-network/omni/blob/main/contracts/src/protocol/XRegistryReplica.sol) on other chains."